Test/git-issues/git-issue-1903.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method g<T>(x : seq<T> := []) 
+{
+   print "worked!\n";  
+}
+
+
+method {:verify true} Main() { 
```

Comment:
```suggestion
method Main() { 
```

Please confirm if the change I suggest is relevant. It seems that verify is always true by default.

Reply:
That is true, I was using example provided in the issue thus didn't remove it I did that now.